src/Air.zig

```diff
@@ -1879,14 +1880,6 @@ pub fn mustLower(air: Air, inst: Air.Inst.Index, ip: *const InternPool) bool {
         .work_group_id,
         => false,
 
-        .assembly => {
-            const extra = air.extraData(Air.Asm, data.ty_pl.payload);
-            const is_volatile = @as(u1, @truncate(extra.data.flags >> 31)) != 0;
-            return is_volatile or if (extra.data.outputs_len == 1)
-                @as(Air.Inst.Ref, @enumFromInt(air.extra[extra.end])) != .none
-            else
-                extra.data.outputs_len > 1;
-        },
```

Comment:
I don't think this is correct. Inline assembly can return a value without there being an output, and in such case it does not need to be marked volatile.

I failed to identify this regression when merging ec36ac8b21fa6c6d8514c60555aba8a96490a560.

Reply:
Can you clarify what that would look like?

Reply:
@andrewrk I'm guessing you're referring to "result" style output constraints (like `[ret] "={rax}" (-> usize)`)? If so, you're misunderstanding the AIR representation here (understandable, since it's horribly documented). Those constraints are *included* in `outputs_len`.

If you're not referring to that, I'm just as confused as @alexrp :P